liblangutil/ParserBase.cpp

```diff
@@ -126,7 +126,7 @@ void ParserBase::expectTokenOrConsumeUntil(Token _value, string const& _currentN
 	}
 
 	if (_advance)
-		m_scanner->next();
```

Comment:
How come these were still used? We did replace many of these in the past.

Reply:
I have no idea. But now there are definitely no uses of it anymore. (Sorry for the late reply, again, I'm catching up with my GH notifications right now ^^)